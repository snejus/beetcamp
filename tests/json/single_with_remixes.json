{
  "@context": "https://schema.org",
  "@id": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 287692048
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 1
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 1858109989
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 3338267481
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 287692048
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a0287692048_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "Emotion 1 - Kulør 008",
      "recordLabel": {
        "@id": "https://kulorco.bandcamp.com",
        "@type": "MusicGroup",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "url_suffix",
            "value": "?from=btl"
          }
        ],
        "name": "Kulør"
      }
    },
    {
      "@id": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008#p1985399228",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 1985399228
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "p"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 3338267481
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Vinyl LP"
        },
        {
          "@type": "PropertyValue",
          "name": "image_ids",
          "value": [
            23426665
          ]
        },
        {
          "@type": "PropertyValue",
          "name": "is_music_merch",
          "value": true
        },
        {
          "@type": "PropertyValue",
          "name": "type_id",
          "value": 2
        }
      ],
      "description": "Vinyl is expected later in March.\nPostage price includes tracking\n\nPlease be aware that we are still experiencing covid related issues with shipping companies. The package might therefore take longer than usual to arrive",
      "image": [
        "https://f4.bcbits.com/img/0023426665_10.jpg"
      ],
      "musicReleaseFormat": "VinylFormat",
      "name": "Emotion 1 Vinyl",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "includes_digital_download",
            "value": true
          }
        ],
        "availability": "InStock",
        "price": 105.0,
        "priceCurrency": "DKK",
        "priceSpecification": {
          "minPrice": 105.0
        },
        "url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008#p1985399228-buy"
      }
    },
    {
      "@id": "https://reececox.bandcamp.com/track/emotion-1",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://reececox.bandcamp.com/track/emotion-1-ibons-dizzy-stomp-mix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://reececox.bandcamp.com/track/emotion-1-parris-remix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://reececox.bandcamp.com/track/emotion-1-call-super-remix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://reececox.bandcamp.com/track/emotion-1-upsammy-remix",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@id": "https://reececox.bandcamp.com",
    "@type": "MusicGroup",
    "name": "Reece Cox"
  },
  "copyrightNotice": "All Rights Reserved",
  "creditText": "W/P by Reece Cox \r\nMastered by Joel Krozer, Six Bit Deep\r\nCreative direction: Najaaraq Vestbirk \r\nArtwork: Holler in the Green by Miriam Kongstad \r\nGraphic design: Spine Studio \r\n\r\nA1 Reece Cox - Emotion 1 (4:55) \r\nWritten and produced by Reece Cox. \r\n\r\nA2 Reece Cox - Emotion 1 (Ibon's Dizzy Stomp Mix) (8:44) \r\nRemix by DJ Ibon.\r\n\r\nB1 Reece Cox - Emotion 1 (Parris Remix) (5:40) \r\nRemix by Dwayne Parris-Robinson, mixed by Call Super. \r\n\r\nB2 Reece Cox - Emotion 1 (Call Super Remix) (6.40) \r\nRemix by Call Super. \r\n\r\nB3 Reece Cox - Emotion 1 (Upsammy Remix) (4:26) \r\nRemix by Thessa Torsing.",
  "dateModified": "09 Apr 2021 07:41:27 GMT",
  "datePublished": "05 Mar 2021 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a0287692048_10.jpg",
  "keywords": [
    "Electronic",
    "Berlin"
  ],
  "mainEntityOfPage": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
  "name": "Emotion 1 - Kulør 008",
  "numTracks": 5,
  "publisher": {
    "@id": "https://reececox.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 3937885934
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_download_codes",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_policies",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 700
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 28280113
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 700
      }
    ],
    "foundingLocation": {
      "@type": "Place",
      "name": "Berlin, Germany"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0028280113_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebSite",
        "name": "reececox.info",
        "url": "http://reececox.info"
      },
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/23x823ddj9iik"
      }
    ],
    "name": "Reece Cox",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://reececox.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "merch",
        "url": "https://reececox.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://reececox.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://reececox.bandcamp.com/track/emotion-1",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 4095985028
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M55S",
          "mainEntityOfPage": "https://reececox.bandcamp.com/track/emotion-1",
          "name": "Emotion 1"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://reececox.bandcamp.com/track/emotion-1-ibons-dizzy-stomp-mix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2433903511
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H08M41S",
          "mainEntityOfPage": "https://reececox.bandcamp.com/track/emotion-1-ibons-dizzy-stomp-mix",
          "name": "Emotion 1 (Ibon's Dizzy Stomp Mix)"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://reececox.bandcamp.com/track/emotion-1-parris-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3231912467
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M39S",
          "mainEntityOfPage": "https://reececox.bandcamp.com/track/emotion-1-parris-remix",
          "name": "Emotion 1 (Parris Remix)"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://reececox.bandcamp.com/track/emotion-1-call-super-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1217738624
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M40S",
          "mainEntityOfPage": "https://reececox.bandcamp.com/track/emotion-1-call-super-remix",
          "name": "Emotion 1 (Call Super Remix)"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://reececox.bandcamp.com/track/emotion-1-upsammy-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1473625405
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M25S",
          "mainEntityOfPage": "https://reececox.bandcamp.com/track/emotion-1-upsammy-remix",
          "name": "Emotion 1 (upsammy Remix)"
        },
        "position": 5
      }
    ],
    "numberOfItems": 5
  }
}
