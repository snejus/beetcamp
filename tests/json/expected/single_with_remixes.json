[
  {
    "album": "Emotion 1",
    "album_id": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": [
      "album",
      "single"
    ],
    "artist": "Reece Cox",
    "artist_credit": null,
    "artist_id": "https://reececox.bandcamp.com",
    "artist_sort": null,
    "artists": [
      "Reece Cox"
    ],
    "artists_credit": [],
    "artists_ids": [
      "https://reececox.bandcamp.com"
    ],
    "artists_sort": [],
    "asin": null,
    "barcode": null,
    "catalognum": "Kulør 008",
    "comments": "W/P by Reece Cox \nMastered by Joel Krozer, Six Bit Deep\nCreative direction: Najaaraq Vestbirk \nArtwork: Holler in the Green by Miriam Kongstad \nGraphic design: Spine Studio \n\nA1 Reece Cox - Emotion 1 (4:55) \nWritten and produced by Reece Cox. \n\nA2 Reece Cox - Emotion 1 (Ibon's Dizzy Stomp Mix) (8:44) \nRemix by DJ Ibon.\n\nB1 Reece Cox - Emotion 1 (Parris Remix) (5:40) \nRemix by Dwayne Parris-Robinson, mixed by Call Super. \n\nB2 Reece Cox - Emotion 1 (Call Super Remix) (6.40) \nRemix by Call Super. \n\nB3 Reece Cox - Emotion 1 (Upsammy Remix) (4:26) \nRemix by Thessa Torsing.",
    "country": "DE",
    "data_source": "bandcamp",
    "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
    "day": 5,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": null,
    "label": "Kulør",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 3,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "release_group_title": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 295,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1",
        "track_alt": null,
        "track_id": "https://reececox.bandcamp.com/track/emotion-1",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 521,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Ibon's Dizzy Stomp Mix)",
        "track_alt": null,
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-ibons-dizzy-stomp-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 339,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Parris Remix)",
        "track_alt": null,
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-parris-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 400,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Call Super Remix)",
        "track_alt": null,
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-call-super-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": null,
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 265,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (upsammy Remix)",
        "track_alt": null,
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-upsammy-remix",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2021
  },
  {
    "album": "Emotion 1",
    "album_id": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008#p1985399228",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": [
      "album",
      "single"
    ],
    "artist": "Reece Cox",
    "artist_credit": null,
    "artist_id": "https://reececox.bandcamp.com",
    "artist_sort": null,
    "artists": [
      "Reece Cox"
    ],
    "artists_credit": [],
    "artists_ids": [
      "https://reececox.bandcamp.com"
    ],
    "artists_sort": [],
    "asin": null,
    "barcode": null,
    "catalognum": "Kulør 008",
    "comments": "Vinyl is expected later in March.\nPostage price includes tracking\n\nPlease be aware that we are still experiencing covid related issues with shipping companies. The package might therefore take longer than usual to arrive\n---\nW/P by Reece Cox \nMastered by Joel Krozer, Six Bit Deep\nCreative direction: Najaaraq Vestbirk \nArtwork: Holler in the Green by Miriam Kongstad \nGraphic design: Spine Studio \n\nA1 Reece Cox - Emotion 1 (4:55) \nWritten and produced by Reece Cox. \n\nA2 Reece Cox - Emotion 1 (Ibon's Dizzy Stomp Mix) (8:44) \nRemix by DJ Ibon.\n\nB1 Reece Cox - Emotion 1 (Parris Remix) (5:40) \nRemix by Dwayne Parris-Robinson, mixed by Call Super. \n\nB2 Reece Cox - Emotion 1 (Call Super Remix) (6.40) \nRemix by Call Super. \n\nB3 Reece Cox - Emotion 1 (Upsammy Remix) (4:26) \nRemix by Thessa Torsing.",
    "country": "DE",
    "data_source": "bandcamp",
    "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
    "day": 5,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": null,
    "label": "Kulør",
    "language": null,
    "media": "Vinyl",
    "mediums": 1,
    "month": 3,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "release_group_title": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": "Emotion 1 Vinyl",
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 295,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1",
        "track_alt": "A1",
        "track_id": "https://reececox.bandcamp.com/track/emotion-1",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": "Emotion 1 Vinyl",
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 521,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Ibon's Dizzy Stomp Mix)",
        "track_alt": "A2",
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-ibons-dizzy-stomp-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": "Emotion 1 Vinyl",
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 339,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Parris Remix)",
        "track_alt": "B1",
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-parris-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": "Emotion 1 Vinyl",
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 400,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (Call Super Remix)",
        "track_alt": "B2",
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-call-super-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Emotion 1",
        "arranger": null,
        "artist": "Reece Cox",
        "artist_credit": null,
        "artist_id": "https://reececox.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Reece Cox"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://reececox.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://reececox.bandcamp.com/album/emotion-1-kul-r-008",
        "disctitle": "Emotion 1 Vinyl",
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 265,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 5,
        "release_track_id": null,
        "title": "Emotion 1 (upsammy Remix)",
        "track_alt": "B3",
        "track_id": "https://reececox.bandcamp.com/track/emotion-1-upsammy-remix",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2021
  }
]
